packages/mds-stream/index.ts

```diff
@@ -93,7 +93,7 @@ declare module 'redis' {
 
   interface Multi {
     xadd: (...args: unknown[]) => string
-    execAsync: () => Promise<string[]>
+    execAsync: () => Promise<object[]>
```

Comment:
This was most definitely not returning a `Promise<string[]>`, and previously wasn't caught due to the janky nested `Promise` implementation.